.travis.yml

```diff
@@ -42,6 +55,22 @@ matrix:
         - TESTMODE=full
         - COVERAGE="--coverage --gcov"
         - NUMPYSPEC="numpy==1.15.4"
+    # Test SciPy with the debug version of Python.
+    # However travis only specifies regular or development builds of Python
+    # so we must install python3-dbg using apt.
+    - python: 3.7
```

Comment:
Using ``python3-dbg`` and ``python3-dev`` will install the distribution's default python versions so the actual python version doesn't matter. However I should lower this to avoid using sudo

Reply:
Is sudo still required for python 3.7? Looking at the linked issue (travis-ci/travis-ci#9815) it sounds like sudo was only needed before `dist: xenial` was set as the default on travis.

Reply:
They may still be required. Right now Travis is converting their serves to Xenial but they are still not 100% there. Similarily the top link (http://lint.travis-ci.org/) is deprecated. I really just want to make sure that adding these build changes don't break anything before making more changes.